src/internal/m365/collection/site/restore.go

```diff
@@ -211,7 +211,7 @@ func RestoreListCollection(
 				restoreContainerName,
 				errs)
 			if err != nil &&
-				errors.Is(err, api.ErrCannotCreateWebTemplateExtension) {
+				errors.Is(err, api.ErrCannotCreateNonRestorableListTemplate) {
 				continue
 			}
 
```

Comment:
Can't comment on unchanged code, but we should add a similar `logger.CtxErr` on line 219 below and other places where we are adding a recoverable error or if we have an error that we are ignoring.